xmlwriter.py

```diff
@@ -17,126 +17,131 @@
 http://www.gnu.org/licenses/gpl.html, or write to the Free Software
 Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110,
 USA.
- 
+
 Should a provision of no. 9 and 10 of the GNU General Public License
 be invalid or become invalid, a valid provision is deemed to have been
 agreed upon which comes closest to what the parties intended
 commercially. In any case guarantee/warranty shall be limited to gross
 negligent actions or intended actions or fraudulent concealment.
 """
 
-import codecs, string, types
+import codecs, types, sys
+from functools import reduce
+
 
 class XmlWriter:
-    def __init__(self, file, encoding='ISO-8859-1'):
-	self.path = []
-	self.encoding = encoding
-	encoder, decoder, streamReader, streamWriter = codecs.lookup(encoding)
-	self.file = streamWriter(file)
-	self.margin = 78
+    def __init__(self, file, encoding='UTF-8'):
+        self.path = []
+        self.encoding = encoding
+        encoder, decoder, streamReader, streamWriter = codecs.lookup(encoding)
```

Comment:
this is also weird... the lifetimes(scopes) of these four variables  are limited to __init__, no?

Reply:
The line should be definitely removed, it contains dead code